plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -128,12 +129,14 @@ public function testIndex()
                             'links' => [
                                 'related' => 'http://api.example.com/documents/2/test',
                                 'self' => 'http://api.example.com/documents/2/relationships/test',
+                                'available' => 'http://api.example.com/objects?filter%5Btype%5D%5B0%5D=documents&filter%5Btype%5D%5B1%5D=profiles',
```

Comment:
Since the URL must be encoded I think it is correct. Very ugly, but correct.

From JsonAPI spec:

> Note: The above example URI shows unencoded [ and ] characters simply for readability. In practice, these characters must be percent-encoded, per the requirements in RFC 3986.

We could make the tests more readble writing something as

```php
'available' => sprintf(
    'http://api.example.com/objects?%s',
    http_build_query(['filter' => ['type' => ['documents', 'profiles']]])
)
```

Just a little problem with that. CakePHP uses `http_build_query()` to generate encoded url that by default uses `RFC 1738` that converts spaces with `+` instead of `%20`.